{"name": "Запрос gettradingvolume: в перечне есть несуществующий логин.", "status": "failed", "statusDetails": {"message": "AssertionError: assert ('1' == '1'\n    1 and 2 == 5)\n +  where 5 = len([{'action': '0', 'close_price': '-', 'close_time': '-', 'comment': '', ...}, {'action': '1', 'close_price': '-', 'clos...'test_modifyposition', ...}, {'action': '0', 'close_price': '-', 'close_time': '-', 'comment': '', ...}, {'login': ''}])", "trace": "open_position = {'from': 1604324239, 'position_1': '102868', 'position_2': '102869', 'to': 1604324265}\n\n    @allure.title(\"Запрос gettradingvolume: в перечне есть несуществующий логин.\")\n    @allure.severity('critical')\n    def test_gettradingvolume_4(open_position):\n        \"\"\"\n        Описание шагов:\n            Setup: создаём аккаунты с логином 100,101.\n                - Пополняем баланс аккаунта\n                - Открываем на нём позицию.\n                - Запрашиваем время сервера для значений from и to\n            Тест:\n                Шаг1: отправляем запрос gettradingvolume c несуществующим логином в перечне - 99\n                Шаг2: проверка ответа\n                Ожидаемый ответ: result=1&reason=&users=2&failed_login=99&size=63\n                                login;order;symbol;price;close_price(-);profit;volume;open_time;close_time(-);conversation_price(-);comment;commission;commission_agent(-);action;profit_rate;dealer;entry\n                                &request_id=\n            Tear down:\n                - Закрытие сделки\n                - Удаление аккаунта\n        \"\"\"\n        with allure.step('Запросом gettradingvolume c несуществующим логином в перечне'):\n            gettradingvolume = wik5.request(wik_mt5_addr, 'gettradingvolume', **{'login': \"99;101\", 'from': open_position['from'], 'to': open_position['to']})\n            # Данные для отчёта: stdout\n            print(\"Ответ на запрос gettradingvolume: \", gettradingvolume)\n            with allure.step('Проверка ответа'):\n>               assert \"1\" == gettradingvolume[\"result\"] and 2 == len(gettradingvolume[\"size\"]) and \"99\" == gettradingvolume['failed_logins']\nE               AssertionError: assert ('1' == '1'\nE                   1 and 2 == 5)\nE                +  where 5 = len([{'action': '0', 'close_price': '-', 'close_time': '-', 'comment': '', ...}, {'action': '1', 'close_price': '-', 'clos...'test_modifyposition', ...}, {'action': '0', 'close_price': '-', 'close_time': '-', 'comment': '', ...}, {'login': ''}])\n\nAntelope\\test_gettradingvolume.py:147: AssertionError"}, "description": "\n    Описание шагов:\n        Setup: создаём аккаунты с логином 100,101.\n            - Пополняем баланс аккаунта\n            - Открываем на нём позицию.\n            - Запрашиваем время сервера для значений from и to\n        Тест:\n            Шаг1: отправляем запрос gettradingvolume c несуществующим логином в перечне - 99\n            Шаг2: проверка ответа\n            Ожидаемый ответ: result=1&reason=&users=2&failed_login=99&size=63\n                            login;order;symbol;price;close_price(-);profit;volume;open_time;close_time(-);conversation_price(-);comment;commission;commission_agent(-);action;profit_rate;dealer;entry\n                            &request_id=\n        Tear down:\n            - Закрытие сделки\n            - Удаление аккаунта\n    ", "steps": [{"name": "Запросом gettradingvolume c несуществующим логином в перечне", "status": "failed", "statusDetails": {"message": "AssertionError: assert ('1' == '1'\n    1 and 2 == 5)\n +  where 5 = len([{'action': '0', 'close_price': '-', 'close_time': '-', 'comment': '', ...}, {'action': '1', 'close_price': '-', 'clos...'test_modifyposition', ...}, {'action': '0', 'close_price': '-', 'close_time': '-', 'comment': '', ...}, {'login': ''}])\n", "trace": "  File \"C:\\Users\\Pavel Starovoitov.TFBDEV\\PycharmProjects\\wikmt5_antelope\\Antelope\\test_gettradingvolume.py\", line 147, in test_gettradingvolume_4\n    assert \"1\" == gettradingvolume[\"result\"] and 2 == len(gettradingvolume[\"size\"]) and \"99\" == gettradingvolume['failed_logins']\n"}, "steps": [{"name": "Проверка ответа", "status": "failed", "statusDetails": {"message": "AssertionError: assert ('1' == '1'\n    1 and 2 == 5)\n +  where 5 = len([{'action': '0', 'close_price': '-', 'close_time': '-', 'comment': '', ...}, {'action': '1', 'close_price': '-', 'clos...'test_modifyposition', ...}, {'action': '0', 'close_price': '-', 'close_time': '-', 'comment': '', ...}, {'login': ''}])\n", "trace": "  File \"C:\\Users\\Pavel Starovoitov.TFBDEV\\PycharmProjects\\wikmt5_antelope\\Antelope\\test_gettradingvolume.py\", line 147, in test_gettradingvolume_4\n    assert \"1\" == gettradingvolume[\"result\"] and 2 == len(gettradingvolume[\"size\"]) and \"99\" == gettradingvolume['failed_logins']\n"}, "start": 1604313455589, "stop": 1604313455589}], "start": 1604313455587, "stop": 1604313455589}], "attachments": [{"name": "stdout", "source": "c6bfcc24-32c1-4590-ace5-370f1d191db5-attachment.txt", "type": "text/plain"}], "start": 1604313455587, "stop": 1604313455590, "uuid": "5de9aeed-4614-431d-8bc7-37511bab1ab4", "historyId": "04ec2c213b9060cfb0c1af623d8a1449", "testCaseId": "53a619c56396749ec1e4c188539afd0d", "fullName": "Antelope.test_gettradingvolume#test_gettradingvolume_4", "labels": [{"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "Antelope"}, {"name": "suite", "value": "test_gettradingvolume"}, {"name": "host", "value": "WKST-T-5"}, {"name": "thread", "value": "13344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Antelope.test_gettradingvolume"}]}